tests/unit_tests/harvester/test_download_publication_utils.py

```diff
@@ -16,6 +19,7 @@ def test_wiley_download(self):
         # Given
         urls, local_entry, filename = wiley_parsed_entry
         # When
+        # TODO: need to instantiate a correct wiley client and make sure it is instantiated one time
```

Comment:
I can't see any place where the new code is tested. Could we do that before merge ? Thx

Reply:
we will just make some unit tests here, we will create an integration test for all the harvesters sources (arxiv, wiley and standard) afterward in another ticket